pyatlan/model/assets/core/asset.py

```diff
@@ -3207,31 +3205,31 @@ def asset_policies_count(self, asset_policies_count: Optional[int]):
         self.attributes.asset_policies_count = asset_policies_count
 
     @property
-    def domain_g_u_i_ds(self) -> Optional[Set[str]]:
-        return None if self.attributes is None else self.attributes.domain_g_u_i_ds
+    def domain_guids(self) -> Optional[Set[str]]:
+        return None if self.attributes is None else self.attributes.domain_guids
 
-    @domain_g_u_i_ds.setter
-    def domain_g_u_i_ds(self, domain_g_u_i_ds: Optional[Set[str]]):
+    @domain_guids.setter
+    def domain_guids(self, domain_guids: Optional[Set[str]]):
```

Comment:
Do we really need to change this attribute? 🤔  
- `domain_g_u_i_ds` (I think we should keep it as it is since we follow the `snake_case` convention for attributes. For example:  
  - `ASSET_ANOMALO_DQ_STATUS` (search attribute) → `asset_soda_d_q_status` (attribute)  
  - Similarly, `DOMAIN_GUIDS` (search attribute) → `domain_g_u_i_ds` (attribute)

Reply:
Yup that is true. I will reverse these changes back then ? 🤔

Reply:
Yes, we should, as it will impact custom packages that are currently using this attribute. 🙏 
https://github.com/search?q=repo%3Aatlanhq%2Fmarketplace-csa-scripts%20domain_g_u_i_ds&type=code

Reply:
Done!